### PR TITLE
Demo: auto-select offline copy of asset after storing

### PR DESCRIPTION
**Pull request by beaufortfrancois**
FIX: #996

### CODE REVIEW

File: demo/offline_section.js [hunk 1]
```diff
@@ -201,7 +210,7 @@ shakaDemo.refreshAssetList_ = function() {
     group.removeChild(group.firstChild);
   }
 
-  shakaDemo.setupOfflineAssets_();
+  return shakaDemo.setupOfflineAssets_();
```
Comment (joeyparrish) [inline demo/offline_section.js]: Since this returns a value now, please update the annotations with @return {!Promise}

Comment (beaufortfrancois) [reply demo/offline_section.js]: Done in https://github.com/google/shaka-player/pull/1001/commits/e3af7f17674d23d3a8772a5c1e828b617bd81b81

**Comment by beaufortfrancois** [comment]
@joeyparrish We should be good now ;)

**Comment by joeyparrish** [comment]
Thanks!  Doing a test pass on the build bot now.

**Comment by joeyparrish** [comment]
There are a couple of build failures about `children` vs `childElements`  (`children` attribute is not present on all platforms we support).

I also did some manual testing of your change locally, and it's not working correctly for me.  After storing something, the store button goes gray and the help text says `The asset is stored offline. Checkout the "Offline" section in the "Asset" list`.  The URI hash has changed to `asset=offline:0`, but the original asset is still selected in the drop-down list itself.  Perhaps you're missing some call to update the list to match?

**Comment by joeyparrish** [comment]
Sorry, not `childElements`.  I meant to say `childNodes`.

**Comment by joeyparrish** [comment]
And now it's working for me locally.  Maybe I had something cached.  Sorry for the noise.

**Comment by beaufortfrancois** [comment]
I believe it is now fixed. @joeyparrish WDYT?

**Comment by joeyparrish** [comment]
Looks good.  Running tests now.  Thanks!

**Comment by joeyparrish** [comment]
Merged.  Thanks for your contribution!

**Comment by joeyparrish** [comment]
I am planning to cherry-pick this to v2.2.2.

**Comment by joeyparrish** [comment]
This has been cherry-picked and will be released in v2.2.2.

**Pull request merged**